xml/System.Windows.Forms/Control.xml

```diff
@@ -2078,8 +2078,26 @@
   
  For more information about handling events, see [Handling and Raising Events](~/docs/standard/events/index.md).  
   
-   
+## Notes to inheritors
```

Comment:
Despite the level-2 head, this renders as level-3.

Reply:
I'm aware. I have an ask to the eng team to consider this section and other notes as H2. Not sure when they will fix though. Ok to keep this change?